### PR TITLE
1ES container templates

### PR DESCRIPTION
**Pull request by danieljurek**
Fixes [#7984](https://github.com/Azure/azure-sdk-tools/issues/7984)

* Uses 1ES tasks for building and pushing docker images (authenticates via service connection) 
* Switches `fpm` container to use Mariner

### CODE REVIEW

File: cli/installer/fpm/fpm.Dockerfile [hunk 1]
```diff
@@ -1,10 +1,6 @@
-ARG prefix=''
-ARG base='ubuntu:22.04'
-FROM ${prefix}${base}
+FROM mcr.microsoft.com/cbl-mariner/base/ruby:3
 
-RUN apt update && \ 
-    DEBIAN_FRONTEND=noninteractive apt install --no-install-recommends --no-install-suggests -y ruby binutils rpm && \
-    gem install fpm
+RUN tdnf install -y tar rpm-build && gem install fpm
```
Comment (ellismg) [inline cli/installer/fpm/fpm.Dockerfile:3]: TIL about `tdnf`.  You're a *nix master now, @danieljurek!

File: eng/pipelines/templates/jobs/verify-installers.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
       ${{ insert }}: ${{ parameters.Variables }}
       BaseUrl: http://127.0.0.1:8080
 
-    timeoutInMinutes: 10
+    timeoutInMinutes: 15
```
Comment (ellismg) [inline eng/pipelines/templates/jobs/verify-installers.yml:32]: Just wondering - I have seen one of the verify installer legs time out from time to time, it this to address that?

**Review by ellismg — APPROVED** [review]
LGTM - sorry for the long delay in getting back to you.

File: eng/pipelines/templates/jobs/verify-installers.yml (hunk 2 above)
Comment (danieljurek) [reply eng/pipelines/templates/jobs/verify-installers.yml:32]: It is. The 1espt work has added tasks which means we're more likely to hit that timeout now.

**Pull request merged**